requirements/extras/boto3.txt

```diff
@@ -0,0 +1 @@
+boto3>=1.4.6
```

Comment:
I think we already have a requirement for boto3. Would you mind checking the DynamoDB requirements?

Reply:
There is 3 requirements for boto3 which are SQS, DynamoDB and this PR.
By checking you mean merging 3 files?

Reply:
SQS requirements are at kombu, right? Perhaps we could use a symlink for `extras/s3.txt` and `extras/dynamodb.txt` both pointing at this file `extras/boto3.txt`?

Reply:
Thanks for pointing it out @othalla . Unfortunately I cannot come up with a better solution, that will also work with the current `setup.py`. Let's leave it as is then :smile: !